reqwest-retry/Cargo.toml

```diff
@@ -24,14 +24,14 @@ reqwest = { version = "0.12.0", default-features = false }
 retry-policies = "0.4"
 thiserror = "1.0.61"
 tracing = { version = "0.1.26", optional = true }
+wasmtimer = "0.4.1"
```

Comment:
We need to gate this dependency behind `cfg(target_arch = "wasm32")` like we did before